man/man8/zpool.8

```diff
@@ -909,6 +909,11 @@ The following subcommands are supported:
 Displays a help message.
 .It Xo
 .Nm
+.Fl V -version
+.Xc
+Displays the ZFS version information
```

Comment:
Let's describe this a bit more precisely, since there's potential confusion with the versioning described by "zpool upgrade".  Specifically, this is telling about the software version, not to be confused with the on-disk format.

Reply:
Perhaps:
_Displays the ZFS program and kernel module version information_
?

Reply:
"ZFS program" could refer to the `zfs program` subcommand.  How about `Displays the software version of the zfs userland utility and the zfs kernel module.`  Or maybe instead of `software version`, we could say `software release`?

Reply:
My feeling is `"Displays the software version of the zfs userland utility and the zfs kernel module"` is the better wording and is consistent with the option name.  Though, I'd OK with `"software release"` as well.

Reply:
done